packages/server/src/tests/testsStore.ts

```diff
@@ -0,0 +1,36 @@
+import * as AWS from 'aws-sdk';
+import { isProd } from '../lib/env';
+import { logError } from '../utils/logging';
+
+const stage = isProd ? 'PROD' : 'CODE';
+
+export type ChannelTypes =
+    | 'Epic'
+    | 'EpicAMP'
+    | 'EpicAppleNews'
+    | 'EpicLiveblog'
+    | 'EpicHoldback'
+    | 'Banner1'
+    | 'Banner2'
+    | 'Header';
+
+export const getTests = <T>(channel: ChannelTypes): Promise<T[]> =>
+    queryChannel(channel, stage)
+        .then(tests => (tests.Items ?? []).sort((a, b) => a.priority - b.priority) as T[])
+        .catch(error => {
+            logError(`Error reading tests from Dynamo: ${error.message}`);
+            return [];
+        });
+
+function queryChannel(channel: ChannelTypes, stage: string) {
+    const docClient = new AWS.DynamoDB.DocumentClient({ region: 'eu-west-1' });
+    return docClient
+        .query({
+            TableName: `support-admin-console-channel-tests-${stage.toUpperCase()}`,
+            KeyConditionExpression: 'channel = :channel',
+            ExpressionAttributeValues: {
+                ':channel': channel,
+            },
+        })
+        .promise();
```

Comment:
Do these not come back in some weird dynamo format? e.g `{ name: { S: 'actually the name' } }`

Reply:
oh that's what the document client does for us! That's so great!

Reply:
It is great indeed!